src/Polar/Constraints/line_flow.jl

```diff
@@ -24,46 +24,25 @@ function flow_constraints(polar::PolarForm, cons, vm, va, pbus, qbus)
 end
 
 function flow_constraints_grad!(polar::PolarForm, cons_grad, buffer, weights)
-    T = typeof(buffer.vmag)
     if isa(buffer.vmag, Array)
-        kernel! = accumulate_view!(KA.CPU())
+        kernel! = adj_branch_flow_kernel!(KA.CPU())
```

Comment:
When `polar` is available in the scope, I would suggest defining the kernel directly as
```julia
kernel! = adj_branch_flow_kernel!(polar.device)
```

Reply:
oh nice! indeed!